dynamic-content/helpers.ts

```diff
@@ -55,7 +55,7 @@ function parseRowContent(row: Row) {
 
 function parseContent(acc: Record<string, string>, [key, value]) {
   if (key === "type" && value) value = value.toLowerCase();
-  if (value && value.includes("*")) value = bulletsToNotifyString(value);
+  if (value && value.includes("*")) value = bulletsToArray(value);
```

Comment:
are there any fields that are consistently bulleted? would it be simpler to have "anything in this field should just be a comma separated string, and will be rendered as bullets"?